CONTRIBUTING.md

```diff
@@ -28,3 +28,32 @@ Members of the Contributors team are encouraged to review pull requests that
 have already been reviewed, and pull request contributors are encouraged to seek
 multiple reviews.  Reviews from anyone not on the Contributors team are always
 appreciated and encouraged!
+
+## Reviewing Pull Requests
+
+All mlpack contributors who choose to review and provide feedback on Pull Requests have a responsibility to both the project and the individual making the contribution. 
+
+Reviews and feedback must be helpful, insightful, and geared towards improving the contribution. If there are reasons why you feel the PR should not be merged, explain what those are. Do not expect to be able to block a Pull Request from advancing simply because you say "No" without giving an explanation. Be open to having your mind changed. Be open to working with the contributor to make the Pull Request better.
```

Comment:
I like this, and I think adding this section is useful because it lets contributors know what to expect.  I hope that for the most part we have always lived up to these guidelines.  :smile:

Actually I came across this today: https://www.youtube.com/watch?v=NNXk_WJzyMI
and I wonder if it's useful to either link to it, or reference it somehow. :+1:

Reply:
May be not the best place, but it wouldn't harm though